content/product/design/wildcard_design_system/index.md

```diff
@@ -65,3 +65,19 @@ When a change to an existing component is required, the designer who is proposin
   - Modify the main instance of the component in Wildcard
   - Publish changes with a fitting commit message
   - Add to the "Changelog" layer in the Wildcard Figma file
+
+## User interface patterns
+
+This section will detail user interface patterns not defined in the Wildcard Design System.
```

Comment:
@rrhyne 
Thoughts on having this information live in WDS in Figma, maybe by the getting started page design comp? I agree that this is the best place to put it given our existing team pages, but I personally would not have thought this information lived here (I would have checked Figma or even checked Figma project files for getting started).

Reply:
I think that's because we do not have design documentation anywhere but Figma.

@AlicjaSuska can you comment on where the WDS documentation project is currently?